R/PLN.R

```diff
@@ -105,7 +105,7 @@ PLN_param <- function(
     Omega         = NULL,
     config_post   = list(),
     config_optim  = list(),
-    inception     = NULL     # pretrained PLNfit used as initialization
+    inception     = NULL     # pretrained PLNfit used as initialization,
```

Comment:
why the additional comma if its the last element of the list?

Reply:
oops :)